### ISSUE TITLE
Fixturemanager::editFixtureProperties - changing Universe does not work correctly

### ISSUE THREAD

**Issue by offtools** [mcallegari/qlcplus#462]
Hi,

I've noticed the following. If you cange the universe of an existing fixture in the fixturemanager ("Change fixture properties" Dialog) the change is not correctly handled. 
The fixture is still listed as child the old universe in the tree widget, although the new universe ID is displayed correct. 
Here simply the updateView() call is missing at the end of FixtureManager::editFixtureProperties().

More Fatal:
The SimpleDesk is not updated correctly. 
Simple Dimmers are still shown in the old universe. For more complex fixtures this can cause a
 a crash (assert in consolechannel.cpp:276), when changing from FixtureManager Tab into SimpleDesk.

The problem seems to be in:

simpledesk.cpp:631 
in Mehtod:
void SimpleDesk::slotUniversePageChanged(int page)

(came up with this issue by working on a softpatch dialog)

offtools

**Comment by offtools** [comment]
Hi,

the crash happens because 
Doc::fixtureForAddress method in simpledesk.cpp:612 (SimpleDesk::slotUniversePageChanged(int page)) returns a wrong Fixture (old universe)

This is because of:

void Doc::slotFixtureChanged(quint32 id)

If I change the universe of the fixture, the fixture has two universe channel entries in Doc::m_channels, one with the old and one with new universe address. So simpledesk is getting conused.

offtools

**Comment by offtools** [comment]
Hi,

guess I fixed the problem. Doc::moveFixture is called with the fixture address, not with the universeAdress, soit could not take the universe change in count and SimpleDesk grabs Fixtures from the wrong universe.

Anyway as changes of the fixtures handled by the fixture itself, emitting   by emtting a change Signal which calls Doc::slotFixtureChanged(quint32 id). We dont need the moveFixture method, if the cleanup / removing old keys section is added to   Doc::slotFixtureChanged(quint32 id).

I did it here:

https://github.com/offtools/qlcplus/commit/3150d30789dcda52f7992f4eff69811af42f76ba

Not sure if this  breaks something ellse

offtools
